wazimap_ng/profile/serializers/profile_indicator_serializer.py

```diff
@@ -6,22 +6,27 @@
 from wazimap_ng.profile.models import ProfileIndicator
 
 from .indicator_data_serializer import get_indicator_data
+from .. import models
 
 
 class ProfileIndicatorSerializer(serializers.ModelSerializer):
-    subcategory = serializers.SerializerMethodField()
-    category = serializers.SerializerMethodField()
+  subcategory = serializers.SerializerMethodField()
+  category = serializers.SerializerMethodField()
+  last_updated = serializers.SerializerMethodField()
 
-    def get_category(self, obj):
-        return obj.subcategory.category.name
+  def get_category(self, obj):
+      return obj.subcategory.category.name
 
-    def get_subcategory(self, obj):
-        return obj.subcategory.name
+  def get_subcategory(self, obj):
+      return obj.subcategory.name
 
-    class Meta:
-        model = ProfileIndicator
-        exclude = ["profile", "id"]
-        depth = 2
+  def get_last_updated(self, obj):
+      return obj.updated
+
+  class Meta:
+      model = models.ProfileIndicator
```

Comment:
why are you using `model.ProfileIndicator` when `ProfileIndicator` is already imported.